src/mbgl/tile/geojson_tile.cpp

```diff
@@ -49,52 +49,28 @@ class GeoJSONTileData : public GeometryTileData {
 std::unique_ptr<GeoJSONTileData> convertTile(const mapbox::geojsonvt::Tile& tile) {
     std::shared_ptr<GeoJSONTileLayer> layer;
 
-    if (tile) {
+    if (tile.features.size()) {
```

Comment:
if `(!tile.features.empty()) {` should be faster per https://github.com/mapbox/mapbox-gl-native/pull/1842